packages/neos-ui/src/index.js

```diff
@@ -152,6 +152,32 @@ function * application() {
     });
 
     fetchWithErrorHandling.registerGeneralErrorHandler((message = 'unknown error') => {
+        // Check if the message is a JSON string
+        if (message.indexOf('{') >= 0) {
```

Comment:
usually people write `!== -1` or even `message.includes('{')`, but nevermind

Reply:
What if the Flow's html error message would contain `{` somewhere in the output?

I would first test for `message.includes('Flow-Debug-Exception-Header')`, then in the else block (since there's no need to test at the same time for Flow's error message and json error message at the same time) try to convert it from JSON and in case if it fails then output this fallback `Unknown error message thing`.